frontend/src/Pages/Favorites.tsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import useSWR from 'swr';
+import { OpenContentFavorite, ServerResponseMany } from '@/common';
+import Pagination from '@/Components/Pagination';
+import { AxiosError } from 'axios';
+import FavoriteCard from '@/Components/FavoriteCard';
+import { useAuth } from '@/useAuth';
+
+export default function FavoritesPage() {
+    const { user } = useAuth();
+    const [perPage, setPerPage] = useState(12);
+    const [pageQuery, setPageQuery] = useState(1);
+
+    const { data, error, isLoading } = useSWR<
+        ServerResponseMany<OpenContentFavorite>,
+        AxiosError
+    >(
+        user
+            ? `/api/open-content/favorites?page=${pageQuery}&per_page=${perPage}`
+            : null,
+        { shouldRetryOnError: false }
+    );
+
+    const favorites = data?.data ?? [];
+    const meta = data?.meta;
+
+    const handleSetPerPage = (val: number) => {
+        setPerPage(val);
+        setPageQuery(1);
+    };
+
+    return (
+        <div className="  p-8 ">
+            <h1>My Favorites</h1>
```

Comment:
this is a super small knit picky thing, but could you remove this h1 from here? i saw it was also in the videos page and must have slipped in there while i was reviewing preston's code, but its not super necessary and looks a little weird spacing wise lol

Reply:
also, in the div that has `p-8` could we add `w-full`? It just makes it so that that block will fill the whole screen rather than just a part of it
<img width="1512" alt="Screenshot 2024-11-25 at 11 25 05 AM" src="https://github.com/user-attachments/assets/f39edc7f-9178-420a-8244-c2a9050fe99a">

vs with `w-full`
<img width="1512" alt="Screenshot 2024-11-25 at 11 25 33 AM" src="https://github.com/user-attachments/assets/5cb1bf25-58eb-4ae6-9122-b1e9a3b48ba4">